go.mod

```diff
@@ -5,7 +5,7 @@ go 1.19
 require (
 	github.com/aws/aws-sdk-go-v2/config v1.18.7
 	github.com/aws/aws-sdk-go-v2/service/marketplacemetering v1.13.26
-	github.com/cert-manager/cert-manager v1.10.1
+	github.com/cert-manager/cert-manager v1.11.0-alpha.2
```

Comment:
is this just a test PR to check compatibility with upcoming 1.11.0?

Reply:
yep just testing